pkg/cloudprovider/vsphereparavirtual/controllers/routablepod/ippool/ippool_controller.go

```diff
@@ -59,38 +54,27 @@ type Controller struct {
 // NewController returns a Controller that reconciles ippool
 func NewController(
 	kubeClient kubernetes.Interface,
-	ippoolclientset ippoolclientset.Interface,
-	ippoolInformer ippoolinformers.IPPoolInformer) *Controller {
+	ippoolManager ippoolmanager.IPPoolManager) *Controller {
 
 	eventBroadcaster := record.NewBroadcaster()
 	eventBroadcaster.StartLogging(klog.Infof)
 	eventBroadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: kubeClient.CoreV1().Events("")})
-	recorder := eventBroadcaster.NewRecorder(ippoolscheme.Scheme, corev1.EventSource{Component: controllerName})
+	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: controllerName})
```

Comment:
using default scheme should be well enough right now to generate an event for node with corev1.ObjectReference